langkit/ocaml_api.py

```diff
@@ -253,6 +253,35 @@ def wrap_requires_context(self, type):
             ' (wrapping): {}'.format(type)
         ))
 
+    def have_ctype_view(self, type):
```

Comment:
I suggest to rename to `has_ctype_view`: we check for a single type, so singular would make more sense

Reply:
Sure.. Done